chart/Chart.yaml

```diff
@@ -1,8 +1,11 @@
+# SPDX-License-Identifier: Apache-2.0
+#!BuildTag: elemental/elemental-operator:0.0.0-noversion
+#!BuildTag: elemental/elemental-operator:0.0.0-noversion-build%RELEASE%
```

Comment:
Will be lost on `helm package` and won't be part of the resulting chart when `make chart` is called.